packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -517,42 +519,65 @@ const ShippingMultiAddress = ({
+                                                            addresses.map((addr) => (
+                                                                <option
+                                                                    key={addr.addressId}
+                                                                    value={addr.addressId}
+                                                                >
+                                                                    {addr.firstName} {addr.lastName}{' '}
+                                                                    - {addr.address1},{' '}
+                                                                    {formatMessage(
+                                                                        {
+                                                                            id: 'shipping_multi_address.format.address_line_2',
+                                                                            defaultMessage:
+                                                                                '{city}, {stateCode} {postalCode}'
+                                                                        },
+                                                                        {
+                                                                            city: addr.city,
+                                                                            stateCode:
+                                                                                addr.stateCode ||
+                                                                                '',
+                                                                            postalCode:
+                                                                                addr.postalCode
+                                                                        }
+                                                                    )}
+                                                                </option>
+                                                            ))
+                                                        )}
+                                                    </Select>
+                                                    <Button
+                                                        variant="outline"
+                                                        size="sm"
+                                                        onClick={() => {
+                                                            setShowAddAddressForm((prev) => ({
+                                                                ...prev,
+                                                                [addressKey]: true
+                                                            }))
+                                                        }}
+                                                        leftIcon={<Box as="span">+</Box>}
```

Comment:
Nit: This is not technically an icon, we would use this if it is a svg icon. Since this is a chacter string. Can we include it in the `addNewAddressLabel`?

Reply:
Let's add aria-label for this button